spec/models/admin_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+describe Admin do
+  describe 'admin' do
+    context 'validations' do
+      subject { build(:admin) }
+      it { is_expected.to be_valid }
+      it { is_expected.to validate_presence_of(:email) }
+      it { is_expected.to validate_length_of(:password).is_at_least(8) }
+    end
+
+    context 'password presence validation' do
+      subject { build(:admin, password: nil) }
+      it { is_expected.to validate_presence_of(:password) }
+    end
```

Comment:
```suggestion
    context 'validations' do
      subject { build(:admin, password: password) }
      let(:password) { 'password' }
      
      it { is_expected.to be_valid }
      it { is_expected.to validate_presence_of(:email) }
      it { is_expected.to validate_length_of(:password).is_at_least(8) }
    end

    context 'password presence validation' do
      let(:password) { nil }
      it { is_expected.to validate_presence_of(:password) }
    end
```
The only reason i suggest this change is then `password presence validation` is actually only changing the password and not recreating a seperate admin for this test only.